examples/acados_matlab_octave/test/test_all_examples.m

```diff
@@ -0,0 +1,132 @@
+%
+% Copyright (c) The acados authors.
+%
+% This file is part of acados.
+%
+% The 2-Clause BSD License
+%
+% Redistribution and use in source and binary forms, with or without
+% modification, are permitted provided that the following conditions are met:
+%
+% 1. Redistributions of source code must retain the above copyright notice,
+% this list of conditions and the following disclaimer.
+%
+% 2. Redistributions in binary form must reproduce the above copyright notice,
+% this list of conditions and the following disclaimer in the documentation
+% and/or other materials provided with the distribution.
+%
+% THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+% AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+% IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+% ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+% LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+% CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+% SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+% INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+% CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+% ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+% POSSIBILITY OF SUCH DAMAGE.;
+
+%
+
+clear; clc; close all;
+p = path;  % save current path for loading later
+start_dir = pwd;
+
+% list the examples you would like to test
+targets = {
+%     '../generic_dyn_disc/disc_dyn_example_ocp.m';
+%     '../generic_external_cost/external_cost_example_ocp.m';
+%     '../getting_started/extensive_example_ocp.m';
+%     '../getting_started/minimal_example_closed_loop.m';
+%     '../getting_started/minimal_example_sim.m';
+%     '../getting_started/simulink_example.m';
+%     '../getting_started/simulink_example_advanced.m';
+%     '../linear_mass_spring_model/example_closed_loop.m';
+%     '../linear_mass_spring_model/example_ocp.m';
+%     '../linear_mass_spring_model/example_sim.m';
+%     '../linear_mpc/example_closed_loop.m';
+%     '../lorentz/example_mhe.m';
+%     '../masses_chain_model/example_closed_loop.m';
+%     '../masses_chain_model/example_ocp.m';
+%     '../pendulum_dae/example_closed_loop.m';  % error while reshaping cost.Vz_0
+%     '../pendulum_dae/example_sim.m';
+%     '../pendulum_on_cart_model/example_closed_loop.m';
+%     '../pendulum_on_cart_model/example_ocp.m';
+%     '../pendulum_on_cart_model/example_ocp_custom_hess.m';
+%     '../pendulum_on_cart_model/example_ocp_param_sens.m';  % Unable to resolve the name ocp.t_ocp.eval_param_sens.
+%     '../pendulum_on_cart_model/example_ocp_reg.m';
+%     '../pendulum_on_cart_model/example_sim.m';
+%     '../pendulum_on_cart_model/example_solution_sens_closed_loop.m';  % Unable to resolve the name ocp.t_ocp.eval_param_sens.
```

Comment:
Oh, thanks for reporting this! I was not aware!
I will fix this soon!
https://github.com/acados/acados/issues/1049

Reply:
Maybe we can use this file to run the CI Matlab tests.
Right now we run them via
`cd examples/acados_matlab_octave/test; run_matlab_tests`, see
https://github.com/acados/acados/blob/f3f1a0f75ce889679f782e0720ad944925aedcbc/.github/workflows/full_build.yml#L143

Reply:
You're welcome :) and thanks for the quick fix.
Yes, I was hoping the script I added could be useful for CI since the examples can be tested directly in that case. It should also work for the existing tests, perhaps with some minor modifications.

Reply:
Great!
Switching the simulink/Matlab CI to use your script to your test would be nice, but let's do this in another PR.